### PR TITLE
Initial working version

### PR DESCRIPTION
**Pull request by matej-tomik**

### CODE REVIEW

File: classes.py [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+    try:
+        ebitda: Series = income_statement.loc['EBITDA']
+        gr_results = []
+        for x in range(len(ebitda) - 1):
+            gr_results.append(ebitda.iloc[x] / ebitda.iloc[x + 1] - 1)
+        return sum(gr_results) / len(gr_results)
+    except KeyError:
+        return stock_info.get('earningsGrowth')
+    except ZeroDivisionError:
+        return stock_info.get('earningsGrowth')
```
Comment (krizanfil) [inline classes.py]:
```suggestion
    except (KeyError, ZeroDivisionError) as e:
        return stock_info.get('earningsGrowth')
```

File: main.py [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+                    ),file=f
+                )
+            except:
+                failed_stocks.append([stock[0], stock[1]])
+                print([stock[0], stock[1]])
+    print('stock fail happens when there is no income statement, balance sheet, info or the ticker is invalid')
+    print(failed_stocks)
+
+# if __name__ == '__main__':
+#
```
Comment (krizanfil) [inline main.py]: uncomment and move to the top of the file, please.

Comment (krizanfil) [inline main.py]: Let's try to develop initial main,py runner for the csv that are downloadable from nasdaq screeners here: https://www.nasdaq.com/market-activity/stocks/screener

**Review by krizanfil — COMMENTED** [review]
GJ!

File: my_data_types.py [hunk 3]
```diff
@@ -16,6 +16,36 @@
 BookValuePerShare = float
 
 MainInfo = Tuple[Optional[Sector], Optional[Industry], Optional[Country], Optional[Name], Optional[MarketCapital]]
-DdmData = Optional[Tuple[Dividend, SharesOutstanding, NetIncome, DividendGrowthRate, Capm, GrowthRate]]
-DcfData = Optional[Tuple[EarningsPerShare, Wacc, GrowthRate]]
+DdmAdvanceData = Optional[Tuple[Dividend, SharesOutstanding, NetIncome, DividendGrowthRate, Capm, GrowthRate]]
+DcfAdvanceData = Optional[Tuple[EarningsPerShare, Wacc, GrowthRate]]
 GrahamNumberData = Optional[Tuple[EarningsPerShare, BookValuePerShare]]
+DdmSimpleData = Optional[Tuple[Dividend, DividendGrowthRate, Capm]]
+DcfSimpleData = Optional[Tuple[Dividend, GrowthRate]]
```
Comment (matej-tomik) [inline my_data_types.py]: DcfSimpleData = Optional[Tuple[Dividend, GrowthRate]] je prepsano na DcfSimpleData = Optional[Tuple[PriceOfStock, GrowthRate]] + PriceOfStock: float je taky pridana

Comment (matej-tomik) [reply my_data_types.py]: PriceOfStock = float

**Pull request merged**